src/randoop/operation/FinalInstanceField.java

```diff
@@ -0,0 +1,34 @@
+package randoop.operation;
+
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+import java.util.List;
+
+import randoop.sequence.Variable;
+
+public class FinalInstanceField extends PublicField {
```

Comment:
It would be delightful to add a spot of documentation for the class if possible.